src/sales-and-marketing/index.md

```diff
@@ -1,10 +1,60 @@
+
+We work towards this goal by:
+
+* managing the end-to-end sales process
+* finding new sales opportunities
+* tracking each step from an opportunity through the Sales Trello board. 
+
+We use this in our weekly sales meeting where we present to the wider dxw team. 
+
+This involves:
+
+* identifying new opportunities: through portals, direct invitations and referrals, approaches from partners, frameworks and pre-market engagements for upcoming opportunities
+* assessing new opportunitie: using a range of criteria. For example, whether it's a good fit based on sector experience, value or an existing relationship. We involve colleagues from across dxw in making decisions to bid. 
+
+We manage the process of bidding for new work, this involves:
+
+1. preparing for the bid by creating a Slack channel, determining the bid team (in discussion with the Heads Of), outlining key dates and steps to bid and adding the opportunity to the sales forecast. 
+
+2. communicating with the client’s procurement team via portals or emails this includes asking questions to help write the bid.
+
+3. developing the bid. This typically involves articulating how we’d approach the client’s problems and create value, drafting project approaches and team shapes, creating project budgets, and selecting case studies that show our skills and experience. This is a collaborative process and we build dxw’s sales capability by pairing with delivery colleagues on bid/pitch writing.
+
+4. preparing the team for pitches sometimes we have the opportunity to present our approach to the client. When this is the case, we finalise who the pitch team are, prepare presentations, and organise practice runs.
+
+We aim to create deliverable, exciting proposals that resonate with client teams and ultimately led to projects that have a positive impact on people who use the services we create. 
+
+## We communicate the outcome, and handover to delivery teams
+
+* when we win, we announce the good news and handover to the Delivery and Commercial teams for client onboarding
+* if we don’t win, we request feedback from the supplier and add it to our Trello board or Sales Crib Sheet. We share the feedback with wider dxw 
+
+We learn from each bid by collecting feedback from all stages to make continuous improvements and feed into future bids and wider dxw strategy.
+
+## We manage sales operations which include 
+
+* monitoring the sales inbox for new opportunities and communication with clients
+* inputting new bids into the sales library and crib sheet to refer to and track feedback 
+* keeping up to date with the market, such as new framework launches and changes to social value in procurement
+* attending networking and training events to make connections and keep our knowledge up to date
+
+
+
+
+
+
+---
+title: Sales and marketing team principles
+related_order: 1
+last_reviewed_at: ""
+---
```

Comment:
Looks like this and what follows has been moved to a separate page so it should be deleted from here